core/src/main/java/org/mule/runtime/core/internal/construct/DefaultFlowBuilder.java

```diff
@@ -221,10 +216,11 @@ public CoreEvent process(final CoreEvent event) throws MuleException {
     @Override
     public ReactiveProcessor referenced() {
       return pub -> from(pub)
+          .doOnNext(assertStarted())
           // Insert the incoming event into the flow, routing it through the processing strategy
-          .flatMap(routeThroughProcessingStrategy()
+          .flatMap(routeThroughProcessingStrategyMapper()
```

Comment:
didn't you just deprecated `routeThroughProcessingStrategyMapper()`?

Reply:
yep, but this usage is more tricky to migrate since it depends on how it is called. Will add a TODO for it